### ISSUE TITLE
Exponential Decay (Half Life) is not practical!

### ISSUE THREAD

**Issue by khanklatt** [zugaldia/halflifeinbox#3]
TL;DR: Use Little's Law.
## 

I see what you did there, but it doesn't make sense.

Suppose you have 1024 messages in your inbox and want to get to 0 (or 1) over a course of 10 days... A recommendation to handle 512 messages the first day, and then half that each successive day has two major problems:
1. Who has time to answer/process 512 emails in one day? (if each response takes 1 minute, you're advocating 8.5 hours of emailing)
2. What about the messages that continue to come in over those 10 days?

The bottom line is that this product, to be practical beyond just a fun exercise, shouldn't be called HalfLifeInbox, it should be Little's Law Inbox, and the math you apply should change.

Applied to the generalized problem you're trying to solve:

The long-term average number of email in a stable inbox (E) is equal to the long-term average effective arrival rate, λ, multiplied by the (Palm‑)average time an email spends in the system (T).

And when you look at it like that, then the equation starts to be really interesting to people trying to truly solve this problem. Then the tool could offer up to three modes:
1. Given E and  λ, you can calculate how long it will take to respond to your messages.
2. Given E and T, you can calculate what the effective arrival rate λ is.
3. Given T and λ, you can calculate how many emails you'll have in your inbox.

How this might look as a webapp (in my view) is the following:

You allow a dropdown for a user to estimate the number of emails they get a day (order of magnitude, 1, 10, 100, 1000).

You ask how many messages they have now.

You ask how many messages they would like to reach (and 0 would be an acceptable input).

Then you ask how long they want to take to get where they want to be.

Use cases (i.e. Unit Tests):
- If they get 100 a day, and they have 1000, and they want to get to 0 in 5 days...

Then tell them they need to answer 300 per day for the next 5 days, and then they need to answer 100 per day from there on out.
- If they get 50 a day and they have 250, and they want to get to 10 in 2 days...

Then tell them they need to answer 170 per day for the next two days, and then 10 per day from there on out.

I know it's simple algebra and it's not as sexy as an exponential decay function, but unless you answer emails like uranium exudes alpha particles, it's not practical.
